src/Nowruz/modules/settings/components/password/index.tsx

```diff
@@ -0,0 +1,98 @@
+                                autoComplete="current_password"
+                                register={register}
+                                name="current_password"
+                                type="password"
+                                placeholder="New password"
+                            />
+                        </div>
+                    </div>
+                </div>
+                <div className={css.borderSection}>
+                    <div className='grid grid-cols-1 lg:grid-cols-5 gap-4'>
+                        <label>Confirm Password</label>
+                        <div className='col-span-2'>
+                            <Input id="password" type="password" name="password" register={register} placeholder="password" />
+                        </div>
+                        
+                    </div>
+                </div>
+                <div className={css.borderSection}>
+                    <div className='grid grid-cols-1 lg:grid-cols-5  gap-4 '>
+                        <label>Confirm New Password</label>
+                        <div className='col-span-2'>
+                            <Input
+                                autoComplete="confirm"
+                                register={register}
+                                name="confirm"
+                                type="password"
+                                placeholder="Confirm new password"
+                            />
+                        </div>
+                    </div>
+                </div>
+                <div>
+                    <div className={`${css.validation} mt-4`}>
+                        <img
+                        className="mr-1"
+                        src={isPasswordLengthValid ? '/icons/green-check.svg' : '/icons/grey-check.svg'}
+                        alt="check"
+                        />
+                        Must be at least 8 characters
+                    </div>
+                    <div className={`${css.validation} mt-2`}>
+                        <img
+                        className="mr-1"
+                        src={isPasswordPatternValid ? '/icons/green-check.svg' : '/icons/grey-check.svg'}
+                        alt="check"
+                        />
+                        Must contain one special character
+                    </div>
+                    <div className={`${css.validation} mt-2 mb-4`}>
+                        <img
+                        className="mr-1"
+                        src={isPasswordMatch ? '/icons/green-check.svg' : '/icons/grey-check.svg'}
+                        alt="check"
+                        />
+                        Password match
+                    </div>
+                </div>
+                <div className="grid grid-cols-1 gap-4 place-items-end pt-8">
+                    <div className='flex gap-4'>
+                        <Button color="info" onClick={()=> reset()}>Cancel</Button>
```

Comment:
suggestion: onClick={reset}

Reply:
i tried your suggestion and it didn't work.